builtin/logical/pki/cert_util.go

```diff
@@ -180,8 +180,8 @@ func fetchCertBySerial(ctx context.Context, req *logical.Request, prefix, serial
 // If one does not pass, it is returned in the string argument.
 func validateNames(b *backend, data *inputBundle, names []string) string {
 	for _, name := range names {
-		sanitizedName := name
-		emailDomain := name
+		sanitizedName := strings.ToLower(name)
```

Comment:
Hello, 
It seems like ToLower doesn't account for non-latin characters, and IDNs can have these characters. Can we instead use EqualFold (https://golang.org/pkg/strings/#EqualFold) in the string comparison? 
Thanks!